2022-08-17T02:40:58.453713+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
2022-08-17T02:40:58.453713+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
2022-08-17T02:40:58.453713+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
2022-08-17T02:40:58.453713+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 846, in exec_module
2022-08-17T02:40:58.453713+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 983, in get_code
2022-08-17T02:40:58.453714+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
2022-08-17T02:40:58.453714+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
2022-08-17T02:40:58.453714+00:00 app[web.1]: File "/app/app.py", line 104
2022-08-17T02:40:58.453714+00:00 app[web.1]: "width": 800,
2022-08-17T02:40:58.453715+00:00 app[web.1]: ^
2022-08-17T02:40:58.453715+00:00 app[web.1]: SyntaxError: invalid syntax
2022-08-17T02:40:58.453783+00:00 app[web.1]: [2022-08-17 02:40:58 +0000] [10] [INFO] Worker exiting (pid: 10)
2022-08-17T02:40:58.469349+00:00 app[web.1]: [2022-08-17 02:40:58 +0000] [4] [WARNING] Worker with pid 10 was terminated due to signal 15
2022-08-17T02:40:58.568316+00:00 app[web.1]: [2022-08-17 02:40:58 +0000] [4] [INFO] Shutting down: Master
2022-08-17T02:40:58.568357+00:00 app[web.1]: [2022-08-17 02:40:58 +0000] [4] [INFO] Reason: Worker failed to boot.
2022-08-17T02:40:58.691639+00:00 heroku[web.1]: Process exited with status 3
2022-08-17T02:40:58.745925+00:00 heroku[web.1]: State changed from starting to crashed
2022-08-17T02:41:47.000000+00:00 app[api]: Build started by user 326ksh@gmail.com
2022-08-17T02:42:18.653778+00:00 app[api]: Deploy 2e434a83 by user 326ksh@gmail.com
2022-08-17T02:42:18.653778+00:00 app[api]: Release v28 created by user 326ksh@gmail.com
2022-08-17T02:42:18.920667+00:00 heroku[web.1]: State changed from crashed to starting
2022-08-17T02:42:23.787544+00:00 heroku[web.1]: Starting process with command `gunicorn app:app`
2022-08-17T02:42:25.114061+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [INFO] Starting gunicorn 20.1.0
2022-08-17T02:42:25.114363+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [INFO] Listening at: http://0.0.0.0:41681 (4)
2022-08-17T02:42:25.114406+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [INFO] Using worker: sync
2022-08-17T02:42:25.117806+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [9] [INFO] Booting worker with pid: 9
2022-08-17T02:42:25.121838+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [9] [ERROR] Exception in worker process
2022-08-17T02:42:25.121838+00:00 app[web.1]: Traceback (most recent call last):
2022-08-17T02:42:25.126134+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
2022-08-17T02:42:25.126135+00:00 app[web.1]: worker.init_process()
2022-08-17T02:42:25.126136+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
2022-08-17T02:42:25.126136+00:00 app[web.1]: self.load_wsgi()
2022-08-17T02:42:25.126136+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2022-08-17T02:42:25.126136+00:00 app[web.1]: self.wsgi = self.app.wsgi()
2022-08-17T02:42:25.126137+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
2022-08-17T02:42:25.126137+00:00 app[web.1]: self.callable = self.load()
2022-08-17T02:42:25.126137+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2022-08-17T02:42:25.126138+00:00 app[web.1]: return self.load_wsgiapp()
2022-08-17T02:42:25.126138+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2022-08-17T02:42:25.126138+00:00 app[web.1]: return util.import_app(self.app_uri)
2022-08-17T02:42:25.126138+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
2022-08-17T02:42:25.126138+00:00 app[web.1]: mod = importlib.import_module(module)
2022-08-17T02:42:25.126139+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
2022-08-17T02:42:25.126139+00:00 app[web.1]: return _bootstrap._gcd_import(name[level:], package, level)
2022-08-17T02:42:25.126139+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
2022-08-17T02:42:25.126139+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
2022-08-17T02:42:25.126139+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
2022-08-17T02:42:25.126140+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
2022-08-17T02:42:25.126140+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 846, in exec_module
2022-08-17T02:42:25.126140+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 983, in get_code
2022-08-17T02:42:25.126140+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
2022-08-17T02:42:25.126140+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
2022-08-17T02:42:25.126141+00:00 app[web.1]: File "/app/app.py", line 104
2022-08-17T02:42:25.126141+00:00 app[web.1]: "width": 800,
2022-08-17T02:42:25.126141+00:00 app[web.1]: ^
2022-08-17T02:42:25.126141+00:00 app[web.1]: SyntaxError: invalid syntax
2022-08-17T02:42:25.126146+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [9] [INFO] Worker exiting (pid: 9)
2022-08-17T02:42:25.128928+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [10] [INFO] Booting worker with pid: 10
2022-08-17T02:42:25.132940+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [10] [ERROR] Exception in worker process
2022-08-17T02:42:25.132941+00:00 app[web.1]: Traceback (most recent call last):
2022-08-17T02:42:25.132942+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
2022-08-17T02:42:25.132943+00:00 app[web.1]: worker.init_process()
2022-08-17T02:42:25.132943+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
2022-08-17T02:42:25.132943+00:00 app[web.1]: self.load_wsgi()
2022-08-17T02:42:25.132943+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2022-08-17T02:42:25.132944+00:00 app[web.1]: self.wsgi = self.app.wsgi()
2022-08-17T02:42:25.132944+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
2022-08-17T02:42:25.132944+00:00 app[web.1]: self.callable = self.load()
2022-08-17T02:42:25.132945+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2022-08-17T02:42:25.132945+00:00 app[web.1]: return self.load_wsgiapp()
2022-08-17T02:42:25.132945+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2022-08-17T02:42:25.132946+00:00 app[web.1]: return util.import_app(self.app_uri)
2022-08-17T02:42:25.132946+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
2022-08-17T02:42:25.132946+00:00 app[web.1]: mod = importlib.import_module(module)
2022-08-17T02:42:25.132946+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
2022-08-17T02:42:25.132947+00:00 app[web.1]: return _bootstrap._gcd_import(name[level:], package, level)
2022-08-17T02:42:25.132947+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
2022-08-17T02:42:25.132948+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
2022-08-17T02:42:25.132948+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
2022-08-17T02:42:25.132948+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
2022-08-17T02:42:25.132949+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 846, in exec_module
2022-08-17T02:42:25.132949+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 983, in get_code
2022-08-17T02:42:25.132949+00:00 app[web.1]: File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
2022-08-17T02:42:25.132950+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
2022-08-17T02:42:25.132950+00:00 app[web.1]: File "/app/app.py", line 104
2022-08-17T02:42:25.132950+00:00 app[web.1]: "width": 800,
2022-08-17T02:42:25.132951+00:00 app[web.1]: ^
2022-08-17T02:42:25.132951+00:00 app[web.1]: SyntaxError: invalid syntax
2022-08-17T02:42:25.133040+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [10] [INFO] Worker exiting (pid: 10)
2022-08-17T02:42:25.152028+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [WARNING] Worker with pid 10 was terminated due to signal 15
2022-08-17T02:42:25.250987+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [INFO] Shutting down: Master
2022-08-17T02:42:25.251012+00:00 app[web.1]: [2022-08-17 02:42:25 +0000] [4] [INFO] Reason: Worker failed to boot.
2022-08-17T02:42:25.396865+00:00 heroku[web.1]: Process exited with status 3
2022-08-17T02:42:25.546734+00:00 heroku[web.1]: State changed from starting to crashed
2022-08-17T02:42:34.000000+00:00 app[api]: Build succeeded
2022-08-17T02:42:57.150528+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=heroku-postgres-son.herokuapp.com request_id=55979d3a-965e-464f-bbc9-605c9cb4bfb0 fwd="58.72.151.126" dyno= connect= service= status=503 bytes= protocol=https
2022-08-17T02:43:48.384256+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=heroku-postgres-son.herokuapp.com request_id=7fd8e93f-80fc-481c-bc08-3cbdfebc6f7a fwd="58.72.151.126" dyno= connect= service= status=503 bytes= protocol=https
2022-08-17T02:44:06.791689+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/speacial" host=heroku-postgres-son.herokuapp.com request_id=26a0a099-c465-4f82-9e81-50b4f1176b99 fwd="58.72.151.126" dyno= connect= service= status=503 bytes= protocol=https
2022-08-17T02:44:08.924111+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=heroku-postgres-son.herokuapp.com request_id=118fd51f-598b-42bb-8451-e4a664260a3e fwd="58.72.151.126" dyno= connect= service= status=503 bytes= protocol=https
2022-08-17T02:44:40.332507+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=heroku-postgres-son.herokuapp.com request_id=4ed4fc5d-f1fc-4747-b44f-fc9ca4662150 fwd="58.72.151.126" dyno= connect= service= status=503 bytes= protocol=https
